concourse/pipeline.yml

```diff
@@ -1878,16 +1878,125 @@ jobs:
       CLOUDFOUNDRY_PASSWORD: ((cloudfoundry_password))
       CLOUDFOUNDRY_ORG: rmras
       CLOUDFOUNDRY_SPACE: ci
-  - task: acceptance-test
+      MAKE_TARGET: rasrm_acceptance_tests
+  - task: acceptance-test-setup
+    on_failure:
+      put: notify
+      params:
+        text: "Acceptance tests setup failed in CI space"
+    privileged: true
+    file: ras-deploy/tasks/rasrm-acceptance-tests/run_acceptance_tests.yml
+    params:
+      <<: *ci_security_user
+      <<: *ci_service_endpoints_for_python
+      MAKE_TARGET: setup
+  - task: rasrm-acceptance-tests
+    on_failure:
+      put: notify
+      params:
+        text: "RASRM acceptance tests failed in CI space"
+    privileged: true
+    file: ras-deploy/tasks/rasrm-acceptance-tests/run_acceptance_tests.yml
+    params:
+      <<: *ci_security_user
+      <<: *ci_service_endpoints_for_python
+      MAKE_TARGET: rasrm_acceptance_tests
+
+- name: ci-securemessage-acceptance-tests
+  serial_groups:
+    *ci_all_services
```

Comment:
If there ins't a serial group with the acceptance tests will both tests kick off at the same time?

Reply:
Because both of the acceptance test jobs belong to the ci_all_services group then only 1 of them will kick off at a time.  Tested that yesterday and it all works.